process/elasticproc/transactions/transactionDBBuilder.go

```diff
@@ -137,7 +138,7 @@ func (dtb *dbTransactionBuilder) prepareTransaction(
 	eTx.Function = converters.TruncateFieldIfExceedsMaxLength(res.Function)
 	eTx.Tokens = converters.TruncateSliceElementsIfExceedsMaxLength(res.Tokens)
 	eTx.ReceiversShardIDs = res.ReceiversShardID
-	eTx.IsRelayed = res.IsRelayed || isRelayedV3
+	eTx.IsRelayed = isRelayedV3
```

Comment:
Does this work all right if we start a post-Barnard node, and sync it from the Genesis, with an indexer attached?

Reply:
applied a better backwards compatible solution